spec/models/ability_spec.rb

```diff
@@ -2,36 +2,25 @@
 require 'cancan/matchers'
 
 describe Ability do
+  let(:subject) { Ability.new(user) }
+  let(:comment) { Comment.new(user: user) }
+
   describe 'for regular user' do
-    let(:user) { FactoryGirl.create(:user) }
+    let(:user) { create(:user) }
 
-      it 'allows to crud comments' do
-        ability = Ability.new(user)
-        comment = Comment.new(user: user)
-        expect(ability).to be_able_to(:manage, comment)
-      end
-
-      it 'can not destroy user' do
-        ability = Ability.new(user)
-        comment = Comment.new(user: user)
-        expect(ability).to_not be_able_to(:destroy, user)
-      end
+      it { is_expected.to be_able_to(:manage, comment) }
+      it { is_expected.to_not be_able_to(:destroy, user) }
 
       it 'can not manage other user comments' do
         second_user = User.new
-        ability = Ability.new(user)
         comment = Comment.new(user: second_user)
-        expect(ability).to_not be_able_to(:manage, comment)
+        is_expected.to_not be_able_to(:manage, comment)
       end
   end
 
   describe 'for admin' do
-    let(:user) { FactoryGirl.create(:user, role: 'admin') }
+    let(:user) { create(:user, role: 'admin') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.